lesson11/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="reset.css" type="text/css" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
+  </head>
+  <body>
+    <div id="js-loading">
+    </div>
+    <div>
+      <ul id="js-list"></ul>
+    </div>
+  </body>
+  <script src="script.js"></script>
```

Comment:
IMO

Declaring `<script>` with 'defer' in `<head> `allows the JS file to be loaded before HTML parsing is complete.
Try it if you like!
[<script> タグに async / defer を付けた場合のタイミング](https://qiita.com/phanect/items/82c85ea4b8f9c373d684)
```suggestion
    <link rel="stylesheet" href="reset.css" type="text/css" />
    <link rel="stylesheet" href="style.css" type="text/css" />
    <script src="script.js" defer></script>
    <title>TerraceTechフロントエンドエンジニア養成所</title>
  </head>
```

Reply:
Thank you for your comment!
I didn't know script defer.
Thank you for teaching me.
I try it defer!

I also noticed that I intended to add the <script> tag just before the </ body> tag, but immediately after that.